flake-rule/src/test/java/com/palantir/flake/example/BeforeAndAfterTest.java

```diff
@@ -36,10 +36,10 @@ public class BeforeAndAfterTest {
     private static final AtomicInteger attemptCount = new AtomicInteger();
     private static final int SEVENTY_SEVEN = 77;
 
-    private static Runnable beforeRunnable = mock(Runnable.class);
-    private static Runnable afterRunnable = mock(Runnable.class);
-    private static Runnable beforeClassRunnable = mock(Runnable.class);
-    private static Runnable afterClassRunnable = mock(Runnable.class);
+    private static Runnable beforeRunnable = spy(new NoOpRunnable());
+    private static Runnable afterRunnable = spy(new NoOpRunnable());
+    private static Runnable beforeClassRunnable = spy(new NoOpRunnable());
+    private static Runnable afterClassRunnable = spy(new NoOpRunnable());
```

Comment:
what baseline check was this failing, out of curiosity?

Reply:
also, this test class is extremely cursed

Reply:
A new one: https://errorprone.info/bugpattern/DirectInvocationOnMock 

It's a _feature_ in scenarios like this (we want to more/less get/set an arbitrary function) we run at odds (not anti-pattern case), but seeing as the change here was minimal, I went ahead and made it.

Reply:
Lambda's cannot be used as they're final, thus you cannot `spy` them